### PR TITLE
chore: first docs cleanup

### PR DESCRIPTION
**Pull request by johanneswuerbach**
Improving the docs.

### CODE REVIEW

**Review by mateuszjenek — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,22 +6,27 @@
 
 ---
 
-A collection of AWS resources ready to be used with [Humanitec](https://humanitec.com/).
+The AWS Resource Pack contains various reference implementation of connecting AWS with [Humanitec](https://humanitec.com/).
```
Comment (TobiasBabin) [inline README.md]: I suggest being more specific here and name the product, i.e. "the Humanitec Platform Orchestrator". "Humanitec" is the name of the company and no longer synonymous with the Orchestrator now that we're multi-product.

File: README.md [hunk 2]
```diff
@@ -6,22 +6,27 @@
...
+* [iam-policy/s3](./humanitec-resource-defs/iam-policy/s3): IAM policy for for an S3 bucket.
+* [iam-policy/sqs](./humanitec-resource-defs/iam-policy/sqs): IAM policy for an SQS queue.
 * [iam-role/service-account](./humanitec-resource-defs/iam-role/service-account): An assumable IAM role, by a k8s service account.
-* [k8s/service-account](./humanitec-resource-defs/k8s/service-account): A k8s service account linked to an IAM role.
-* [redis/basic](./humanitec-resource-defs/redis/basic): AWS ElastiCache resource.
-* [s3/basic](./humanitec-resource-defs/s3/basic): A basic S3 bucket.
-* [sqs/basic](./humanitec-resource-defs/sqs/basic): A basic SQS queue.
-* [workload/service-account](./humanitec-resource-defs/workload/service-account): A workload service account.
+* [k8s/service-account](./humanitec-resource-defs/k8s/service-account): A `k8s-service-account` linked to an IAM role.
+* [rds/aurora](./humanitec-resource-defs/redis/basic): A `postgres` or `mysql` resource using AWS RDS Aurora.
```
Comment (TobiasBabin) [inline README.md]:
```suggestion
* [rds/aurora](./humanitec-resource-defs/rds/aurora): A `postgres` or `mysql` resource using AWS RDS Aurora.
```

File: README.md [hunk 3]
```diff
@@ -6,22 +6,27 @@
...
+* [rds/basic](./humanitec-resource-defs/redis/basic): A `postgres` or `mysql` resource using AWS RDS.
```
Comment (TobiasBabin) [inline README.md]:
```suggestion
* [rds/basic](./humanitec-resource-defs/rds/basic): A `postgres` or `mysql` resource using AWS RDS.
```

File: README.md [hunk 4]
```diff
@@ -6,22 +6,27 @@
...
+* [redis/basic](./humanitec-resource-defs/redis/basic): A `redis` resource using AWS ElastiCache.
+* [s3/basic](./humanitec-resource-defs/s3/basic): A basic `s3` bucket.
+* [sqs/basic](./humanitec-resource-defs/sqs/basic): A basic `sqs` queue.
+* [workload/service-account](./humanitec-resource-defs/workload/service-account): Inject a k8s service account into a `workload`.
 
 The `humanitec-resource-defs` directory includes the respective resource definitions.
```
Comment (TobiasBabin) [inline README.md]:
```suggestion
The `humanitec-resource-defs` directory includes the respective Resource Definitions.
```

File: README.md [hunk 5]
```diff
@@ -6,22 +6,27 @@
...
 
 The `modules` directory includes the Terraform modules used by the [Humanitec Terraform Driver](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/) to provision the actual resources.
 
-Checkout `examples/main.tf` for an example application using S3 & SQS.
+Checkout `examples/` for example application using the resource definitions listed above.
```
Comment (TobiasBabin) [inline README.md]:
```suggestion
Checkout `examples/` for example usage of the Resource Definitions listed above.
```

File: examples/dns/README.md [hunk 6]
```diff
@@ -1,3 +1,16 @@
+# Example: dns resource based on AWS Route 53
+
+This example configures a [dns](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#dns) resource definition using AWS Route 53.
```
Comment (TobiasBabin) [inline examples/dns/README.md]:
```suggestion
This example configures a [dns](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#dns) Resource Definition using AWS Route 53.
```

File: examples/dns/README.md [hunk 7]
```diff
@@ -1,3 +1,16 @@
...
+
+The created resource definition can be used in your score file using:
```
Comment (TobiasBabin) [inline examples/dns/README.md]:
```suggestion
The created Resource Definition can be used in your Score file using:
```

File: examples/mysql/aurora/README.md [hunk 8]
```diff
@@ -1,3 +1,16 @@
+# Example: mysql resource based on AWS RDS Aurora
+
+This example configures a [mysql](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#mysql) resource definition using AWS RDS Aurora.
```
Comment (TobiasBabin) [inline examples/mysql/aurora/README.md]:
```suggestion
This example configures a [mysql](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#mysql) Resource Definition using AWS RDS Aurora.
```

File: examples/mysql/aurora/README.md [hunk 9]
```diff
@@ -1,3 +1,16 @@
...
+
+The created definition can be used in your score file using:
```
Comment (TobiasBabin) [inline examples/mysql/aurora/README.md]:
```suggestion
The created definition can be used in your Score file using:
```

File: examples/mysql/basic/README.md [hunk 10]
```diff
@@ -1,3 +1,16 @@
+# Example: mysql resource based on AWS RDS
+
+This example configures a [mysql](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#mysql) resource definition using AWS RDS.
```
Comment (TobiasBabin) [inline examples/mysql/basic/README.md]:
```suggestion
This example configures a [mysql](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#mysql) Resource Definition using AWS RDS.
```

File: examples/mysql/basic/README.md [hunk 11]
```diff
@@ -1,3 +1,16 @@
...
+
+The created resource definition can be used in your score file using:
```
Comment (TobiasBabin) [inline examples/mysql/basic/README.md]:
```suggestion
The created Resource Definition can be used in your Score file using:
```

File: examples/postgres/aurora/README.md [hunk 12]
```diff
@@ -1,3 +1,16 @@
+# Example: postgres resource based on AWS RDS Aurora
+
+This example configures a [postgres](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#postgres) resource definition using AWS RDS Aurora.
```
Comment (TobiasBabin) [inline examples/postgres/aurora/README.md]:
```suggestion
This example configures a [postgres](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#postgres) Resource Definition using AWS RDS Aurora.
```

File: examples/postgres/aurora/README.md [hunk 13]
```diff
@@ -1,3 +1,16 @@
...
+
+The created resource definition can be used in your score file using:
```
Comment (TobiasBabin) [inline examples/postgres/aurora/README.md]:
```suggestion
The created Resource Definition can be used in your Score file using:
```

File: examples/postgres/basic/README.md [hunk 14]
```diff
@@ -1,3 +1,16 @@
+# Example: postgres resource based on AWS RDS
+
+This example configures a [postgres](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#postgres) resource definition using AWS RDS.
```
Comment (TobiasBabin) [inline examples/postgres/basic/README.md]:
```suggestion
This example configures a [postgres](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#postgres) Resource Definition using AWS RDS.
```

File: examples/postgres/basic/README.md [hunk 15]
```diff
@@ -1,3 +1,16 @@
...
+
+The created resource definition can be used in your score file using:
```
Comment (TobiasBabin) [inline examples/postgres/basic/README.md]:
```suggestion
The created Resource Definition can be used in your Score file using:
```

File: examples/redis/README.md [hunk 16]
```diff
@@ -1,3 +1,16 @@
+# Example: redis resource based on AWS ElastiCache
+
+This example configures a [redis](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#redis) resource definition using AWS ElastiCache.
```
Comment (TobiasBabin) [inline examples/redis/README.md]:
```suggestion
This example configures a [redis](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#redis) Resource Definition using AWS ElastiCache.
```

File: examples/redis/README.md [hunk 17]
```diff
@@ -1,3 +1,16 @@
...
+
+The created resource definition can be used in your score file using:
```
Comment (TobiasBabin) [inline examples/redis/README.md]:
```suggestion
The created Resource Definition can be used in your Score file using:
```

File: examples/s3/README.md [hunk 18]
```diff
@@ -1,11 +1,11 @@
-# AWS S3
+# Example: s3 resource based on AWS S3
 
-This example configures an S3 bucket resource definition, with two different access policies:
+This example configures a [s3](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#s3) resource definition using AWS S3, with two different access policies:
```
Comment (TobiasBabin) [inline examples/s3/README.md]:
```suggestion
This example configures a [s3](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#s3) Resource Definition using AWS S3, with two different access policies:
```

File: examples/s3/README.md [hunk 19]
```diff
@@ -1,11 +1,11 @@
...
 
 * `basic-admin` (full access)
 * `basic-read-only` (read-only access)
 
-Those resources can be used in your score file like:
+Those resource definitions can be used in your score file using:
```
Comment (TobiasBabin) [inline examples/s3/README.md]:
```suggestion
Those Resource Definitions can be used in your Score file using:
```

File: examples/sqs/README.md [hunk 20]
```diff
@@ -1,16 +1,16 @@
-# AWS SQS
+# Example: sqs resource based on AWS SQS
 
-This example configures an SQS queue resource definition, with two different access policies:
+This example configures a [sqs](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#sqs) resource definition using AWS SQS, with two different access policies:
```
Comment (TobiasBabin) [inline examples/sqs/README.md]:
```suggestion
This example configures a [sqs](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#sqs) Resource Definition using AWS SQS, with two different access policies:
```

File: examples/sqs/README.md [hunk 21]
```diff
@@ -1,16 +1,16 @@
...
 
 * `basic-publisher` (allowed to send messages)
 * `basic-consumer` (allowed to receive messages)
 
-Those resources can be used in your score file like:
+Those resource definitions can be used in your score file using:
```
Comment (TobiasBabin) [inline examples/sqs/README.md]:
```suggestion
Those Resource Definitions can be used in your Score file using:
```

**Review by TobiasBabin — CHANGES_REQUESTED** [review]
A bunch of caps items and one naming suggestion, looks great overall!

**Review by TobiasBabin — APPROVED** [review]
🚀

**Pull request merged**